### PR TITLE
Adding new supported states to Ios_vlans resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Adding new supported states to Ios_vlans resource module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ios_vlans
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

File: plugins/modules/ios_vlans.py [hunk 1]
```diff
@@ -394,10 +430,249 @@
...
+#         {
+#             "mtu": 1500,
+#             "name": "trnet-default",
+#             "shutdown": "enabled",
+#             "state": "active",
+#             "vlan_id": 1005
+#         }
+#     ]
+
+# After state:
```
Comment (GomathiselviS) [inline plugins/modules/ios_vlans.py:512]: Do we have to mention after state for gathered operation? IMO , we donot have to. What do you think?

**Comment by justjais** [comment]
recheck

**Pull request merged**